reference-implementation/lib/writable-stream.js

```diff
@@ -60,6 +60,17 @@ class WritableStream {
 
 exports.WritableStream = WritableStream;
 
+Object.assign(exports, {
+  AcquireWritableStreamDefaultWriter,
+  IsWritableStream,
+  WritableStream,
+  WritableStreamAbort,
+  WritableStreamDefaultWriterClose,
```

Comment:
Why is WritableStreamDefaultWriterClose exported? readable-stream.js doesn't import it.